src/main/java/DukeException.java

```diff
@@ -0,0 +1,31 @@
+public class DukeException extends Exception {
+    /**
+     * Prints out the correct error message based on error type.
+     *
+     * @param errorType Description of error.
+     */
+    public DukeException(String errorType) {
+        String errorMessage = errorType.equals("invalidCommand")
```

Comment:
Might be worth using an enum for `errorType`, because it seems weird that you are comparing strings. Might lead to runtime improvements too.